src/adagrams.js

```diff
@@ -1,8 +1,84 @@
+// Distribution of Letters
+const letterPool = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D','D','D','D',
+                    'E','E','E','E','E','E','E','E','E','E','E','E',
+                    'F','F','G','G','G','H','H','I','I','I','I','I','I','I','I','I',
+                    'J','K','L','L','L','L','M','M','N','N','N','N','N','N',
+                    'O','O','O','O','O','O','O','O','P','P','Q','R','S','S','S','S',
+                    'T','T','T','T','T','T','U','U','U','U','V','V','W','W',
+                    'X','Y','Y','Z']
+
+const scoredWords = {};
+scoredWords['A'] = 1;
```

Comment:
These repeated lines are a bit much... Why not just put it inside the `scoredWords` object to begin with? `const scoredWords = { A: 1, B: 3, C: 3}` etc...?